packages/next/package.json

```diff
@@ -35,6 +35,8 @@
     "script.d.ts",
     "server.js",
     "server.d.ts",
+    "server/testing.js",
+    "server/testing.d.ts",
```

Comment:
We should probably nest them under experimental while we validate these APIs

Reply:
`next/experimental/server/testing` or `next/server/experimental/testing`?

Reply:
We already have `next/experimental/testmode` so maybe `next/experimental/testmode/server`?

Reply:
Done.